src/hotspot/share/prims/jvmtiThreadState.hpp

```diff
@@ -231,6 +231,8 @@ class JvmtiThreadState : public CHeapObj<mtInternal> {
   inline JvmtiEnvThreadState* head_env_thread_state();
   inline void set_head_env_thread_state(JvmtiEnvThreadState* ets);
 
+  static bool _seen_interp_only_mode; // interp_only_mode was requested once
```

Comment:
perhaps: s/requested once/requested at least once/

Reply:
Thank you - fixed now.